### PR TITLE
CLUE 3 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/dimir_strandcatcher.txt [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ManaCost:2 UB UB
+Types:Creature Faerie Rogue
+PT:3/3
+K:Flying
+T:Mode$ AttackersDeclared | AttackingPlayer$ You | Execute$ TrigSurveil | TriggerZones$ Battlefield | TriggerDescription$ Whenever you attack, surveil X, where X is the number of opponents being attacked.
+SVar:TrigSurveil:DB$ Surveil | Amount$ X
+T:Mode$ Phase | Phase$ End of Turn | TriggerZones$ Battlefield | CheckSVar$ Y | SVarCompare$ GE3 | Execute$ TrigDraw | TriggerDescription$ At the beginning of each end step, if three or more cards were put into your graveyard from anywhere other than the battlefield this turn, draw a card.
+SVar:TrigDraw:DB$ Draw
+SVar:X:PlayerCountPropertyYou$OpponentsAttackedThisCombat
+SVar:Y:Count$ThisTurnEntered_Graveyard_from_Library_Card/Plus.CountHand
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/dimir_strandcatcher.txt:11]: YouOwn

File: forge-gui/res/cardsfolder/upcoming/furious_spinesplitter.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Name:Furious Spinesplitter
+ManaCost:2 RG RG
+Types:Creature Ogre Warrior
+PT:3/3
+K:Trample
+T:Mode$ Phase | Phase$ End of Turn | TriggerZones$ Battlefield |  Execute$ TrigPutCounter | TriggerDescription$ At the beginning of your end step, put a +1/+1 counter on CARDNAME for each opponent that was dealt damage this turn.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/furious_spinesplitter.txt:6]: spacing ;)

File: forge-gui/res/cardsfolder/upcoming/furious_spinesplitter.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+SVar:TrigPutCounter:DB$ PutCounter | Defined$ Self | CounterType$ P1P1 | CounterNum$ X
+SVar:X:PlayerCountRegisteredOpponents$HasPropertywasDealtDamageThisTurn
+DeckHints:Ability$Counters
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/furious_spinesplitter.txt:9]: don't think this line is needed

File: forge-gui/res/cardsfolder/upcoming/headliner_scarlett.txt [hunk 4]
```diff
@@ -0,0 +1,14 @@
+Name:Headliner Scarlett
+ManaCost:3 R
+Types:Legendary Creature Human Warlock
+PT:3/3
+K:Haste
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Destination$ Battlefield | Execute$ TrigEffect | TriggerDescription$ When CARDNAME enters the battlefield, creatures target player controls can't block this turn.
+SVar:TrigEffect:DB$ Effect | ValidTgts$ Player | TgtPrompt$ Select target player | AILogic$ Evasion | StaticAbilities$ OppCantBlock | RememberObjects$ Targeted
+SVar:OppCantBlock:Mode$ CantBlockBy | ValidBlocker$ Creature.RememberedPlayerCtrl | Description$ Creatures targeted player controls can't block this turn.
+T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | Execute$ TrigExile | TriggerZones$ Battlefield | TriggerDescription$ At the beginning of your upkeep, exile the top card of your library face down. You may look at and play that card this turn.
+SVar:TrigExile:DB$ ChangeZone | Defined$ TopOfLibrary | Origin$ Library |  RememberChanged$ True | Destination$ Exile | ExileFaceDown$ True | SubAbility$ DBEffect
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/headliner_scarlett.txt:10]: spacing

File: forge-gui/res/cardsfolder/upcoming/headliner_scarlett.txt [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+SVar:DBEffect:DB$ Effect | RememberObjects$ Remembered | StaticAbilities$ STPlay | ForgetOnMoved$ Exile | SubAbility$ DBCleanup
+SVar:STPlay:Mode$ Continuous | MayLookAt$ You | MayPlay$ True | EffectZone$ Command | Affected$ Card.IsRemembered | AffectedZone$ Exile | Description$ You may look at and play that card this turn.
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+Oracle:Haste\nWhen Headliner Scarlett enters the battlefield, creatures target player controls can't block this turn.\nAt the beginning of your upkeep, exile the top card of your library face down. You may look at and play that card this turn.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/headliner_scarlett.txt:14]: don't forget your whitespace lines

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/headliner_scarlett.txt:14]: image

I think this PR is fine ?

Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/headliner_scarlett.txt:14]: Sounds good, the github interface makes it hard to tell ;)

**Review by tool4ever — APPROVED** [review]

**Pull request merged**